sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -1,37 +1,37 @@
 <template>
-  <student-page-container :full-width="true">
+  <student-page-container>
     <template #header>
       <header-navigator
         title="Applications"
         subTitle="Financial Aid Application"
       >
+        <template #buttons>
+          <v-row class="m-0 p-0 float-right">
+            <v-btn
+              color="primary"
+              v-if="!notDraft && !isFirstPage && !submittingApplication"
+              variant="outlined"
+              :loading="savingDraft"
+              @click="saveDraft()"
+            >
+              {{ savingDraft ? "Saving..." : "Save draft" }}</v-btn
+            >
+            <v-btn
+              v-if="!isReadOnly && !isFirstPage"
+              class="ml-2"
+              :disabled="!isLastPage || submittingApplication"
```

Comment:
Does anyone remember why this button is disabled? I would prefer to leave it enabled because can be confusing to the user. The user needs to know that the last tab must be selected to this be enabled.
@ann-aot @dheepak-aot @guru-aot @hellolynn-tbtb @abschwenker 
This does not need to hold the PR but I would like to just remove the condition `!isLastPage`.

Reply:
Ok. Enabled button for all pages.

Reply:
mm..I don't remember. was it added before the "draft feature"?